crates/red_knot_python_semantic/mdtest.py

```diff
@@ -0,0 +1,200 @@
+"""A runner for Markdown-based tests for Red Knot"""
+# /// script
+# requires-python = ">=3.11"
+# dependencies = [
+#     "rich",
```

Comment:
I nearly suggested that we use `termcolor` instead of `rich`, since it's a much smaller library to have as a dependency, but I suppose your use of `console.status` is good enough reason to stick with `rich` 😆

Reply:
Right... I have never used `rich` before and it seems a big bloated, but on the other hand, `uv` is fast :smile: